cluster-autoscaler/main.go

```diff
@@ -243,6 +243,7 @@ var (
 	maxAllocatableDifferenceRatio           = flag.Float64("max-allocatable-difference-ratio", config.DefaultMaxAllocatableDifferenceRatio, "Maximum difference in allocatable resources between two similar node groups to be considered for balancing. Value is a ratio of the smaller node group's allocatable resource.")
 	forceDaemonSets                         = flag.Bool("force-ds", false, "Blocks scale-up of node groups too small for all suitable Daemon Sets pods.")
 	dynamicNodeDeleteDelayAfterTaintEnabled = flag.Bool("dynamic-node-delete-delay-after-taint-enabled", false, "Enables dynamic adjustment of NodeDeleteDelayAfterTaint based of the latency between CA and api-server")
+	enablePodTimeBuffer                     = flag.Bool("pod-time-buffer-enabled", true, "Enables waiting for a time buffer so that the oldest unschedulable pod should be before starting scale up.")
```

Comment:
We should control the entire logic with a flag, not just the "wait for more pods part". What I'd expect to see here is a single flag with two settings:
- Keep the time buffers, don't inject scheduler unprocessed pods in CA
- Drop the time buffest, start processing all unscheduled pods

Reply:
Updated flag name to `ignore-scheduler-processing` & used it in unschedulable pod processing in static autoscaler